docs/pages/enroll-resources/application-access/okta/appendix-a-okta-saml-app.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Appendix A. Okta SAML app
```

Comment:
- Why are we calling this an appendix?
- We should edit the title and description per the style guide.